lib/3scale_toolbox/commands/service_command/service_create.rb

```diff
@@ -0,0 +1,33 @@
+module ThreeScaleToolbox
+  module Commands
+    module ServiceCommand
+      class ServiceCreateSubcommand < Cri::CommandRunner
+        include ThreeScaleToolbox::Command
+
+        def self.command
+          Cri::Command.define do
+            name        'create'
+            usage       'create'
+            summary     'service create'
+            description 'Create a service'
+            runner ServiceCreateSubcommand
+
+            param   :source
+            param   :service_name
+          end
+        end
+
+        def run
+          source = arguments[:source]
+          service_name = arguments[:service_name]
+          client = threescale_client(source)
+          result = client.create_service name: service_name
+          if !result['errors'].nil?
+            abort "Service has not been saved. Errors: #{result['errors']}"
```

Comment:
`raise ThreeScaleToolbox::Error`. 

Let the toolbox exit the process and do some cleanup if it has to

Reply:
why not use `Service.create`??

Reply:
:+1: